README.md

```diff
@@ -128,10 +125,10 @@ but the JSON you receive from the server looks like this:
 }
 ```
 
-you would want that JSON parsing to fail and return `.None` instead of a
-`User` object. That base concept means that when you're dealing with an actual
-`User` object, you can be sure that when you ask for `user.id`, you're going
-to get the type you expect.
+you would want that JSON parsing to fail and return some error state instead of
```

Comment:
"...then ideally, JSON parsing would fail, and you'd get an error state instead of a `User` object`."